pages/faqs/faq-inspector.mdx

```diff
@@ -1,14 +1,14 @@
 ---
 layout: 'guide'
-title: 'FAQ'
+title: 'Inspector FAQ'
```

Comment:
I'm wondering if we should keep this page under the Inspector section?

Reply:
Not sure, we don't actually have the dedicated inspector section anymore

Reply:
Actually I think we are lacking a doc answering the question "What is Avo Inspector?"